src/commands/HBonds.hpp

```diff
@@ -0,0 +1,56 @@
+// cfiles, an analysis frontend for the Chemfiles library
+// Copyright (C) 2015-2016 Guillaume Fraux
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+#ifndef CFILES_HBONDS_HPP
+#define CFILES_HBONDS_HPP
+
+#include <chemfiles.hpp>
+
+#include "Command.hpp"
+#include "utils.hpp"
+
+class HBonds final: public Command {
+public:
+    struct Options {
+        /// Input trajectory
+        std::string trajectory;
+	/// Output
+	std::string outfile;
+        /// Specific format to use with the trajectory
+        std::string format = "";
+        /// Specific steps to use from the trajectory
+        steps_range steps;
+	/// Selection for the donor-acceptor
+	std::string selectionAcceptor;
+	std::string selectionDonor;
+        /// Do we have a custom cell to use?
+        bool custom_cell = false;
+        /// Unit cell to use
+        chemfiles::UnitCell cell;
+        /// Topology file to use
+        std::string topology = "";
+        /// Format to use for the topology file
+        std::string topology_format = "";
+        /// Should we try to guess the topology?
+        bool guess_bonds = false;
+	/// Parameters for donor-acceptor max distance (in angstroms)
+	double distance_parameter = 0.0;
+	/// and for donor-acceptor-hydrogen max angle (in degrees)
+	double angle_parameter = 0.0;
+    };
+
+    HBonds(): selectionAcceptor_("bonds: type(#2) == H"), selectionDonor_("not type H") {}
```

Comment:
Please do not provide defaults both here and in the doctopt `OPTIONS` string. There should be only one source of information to keep it [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself).